plugins/email_service/app/services/service_layer/email_service_services/email.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_String_literal: true
+
+module ServiceLayer
+  module EmailServiceServices
+    # email api implementation
+    module Email
+
+      # Creds = Struct.new(:access, :secret)
+      Creds = Struct.new(:access, :secret, :error)
+      def aws_creds(user_id, filter = {})
+
+        response = elektron_email_service.get("users/#{user_id}/credentials/OS-EC2")
```

Comment:
As I mentioned you should use the methods already defined in Identity. Do not do it from your plugin. See hier:
https://github.com/sapcc/elektra/blob/master/plugins/identity/app/services/service_layer/identity_service.rb#L17
If you need such credentials you should define a method here:
https://github.com/sapcc/elektra/blob/master/plugins/identity/app/services/service_layer/identity_services/user.rb
and called from this controller like
`services.identity.credentials("OS_EC2")`

Reply:
done.